### DISCUSSION TITLE
How to fix the error when i click the start to start the sever

### DISCUSSION THREAD

**Discussion by proxy0128** [thepeacockproject/Peacock#548] [Q&A] [answered]
![Screenshot 2024-12-30 180105](https://github.com/user-attachments/assets/634e65b1-5cd6-4214-86f6-35bbad6df966)

**Comment by proxy0128** [comment]
I'm searching every where to fix the problem but i found nothing to fix this so i ask here

**Comment by suanjiansalt** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/963f528f-b851-46df-898c-942567e71f7e)

It already told you
